### ISSUE TITLE
add support lycoris for stable diffusion?

### ISSUE THREAD

**Issue by cm5kZGV2MjAyM3B1YmdpdGh1YmFjYw** [huggingface/diffusers#3869]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Ideally it would follow the official implementation and only add extra community features as needed.

**Describe alternatives you've considered**
Instead of reinvent the wheel we can use and adapt existing implementation and make it compatible.

**Additional context**
For the popular webui stable diffusion,
They have this repo to use LyCORIS
https://github.com/a2569875/stable-diffusion-webui-composable-lora

If I am not wrong,I think in essence it is just a repo with a collection of algorithm.
If we can create a function with optional parameter that can let user pick which algorithm they need.
It would work?

**Comment by alexblattner** [comment]
you can already use lycoris as lora but the training isn't available. I would argue it should be high on the priority list considering how long ago it came out and how useful it is

**Comment by cm5kZGV2MjAyM3B1YmdpdGh1YmFjYw** [comment]
I tried to simply load.with  pipe.load_lora_weights(".", weight_name=lp,cache_dir="LoRA")
but it throws error,do you know how to solve>
lib\site-packages\diffusers\loaders.py", line 1297, in _convert_kohya_lora_to_diffusers
    raise ValueError("Network alpha is not consistent")
ValueError: Network alpha is not consistent

**Comment by alexblattner** [comment]
my bad then. @patrickvonplaten this is incredibly important. More so than any new pipeline that's been added lately in my opinion. Are there any plans to address lycoris?

**Comment by patrickvonplaten** [comment]
Why is this incredibly important? The repo: https://github.com/a2569875/stable-diffusion-webui-composable-lora has ~80 stars.

cc @sayakpaul here for LoRA. 

I currently don't see why this is important though

**Comment by sayakpaul** [comment]
As discussed in https://github.com/huggingface/diffusers/issues/3087, we'll continue to revisit this support.

LyCORIS in a1111 has an array of formats, and it's practically quite challenging for us to support all of them without introducing complexity to our codebase.

**Comment by alexblattner** [comment]
@patrickvonplaten because it's a staple now on civitai. With all due respect, as of now they are the standard setters in the diffusion field.

@sayakpaul I see. You did do something with compel for complex prompt which is external. Maybe something like that could be an option. Good luck with the implementation!

**Comment by jfischoff** [comment]
@patrickvonplaten I'll second what @alexblattner said. Some of the best fine-tuners on Civitai have moved to lychoris exclusively, e.g. https://civitai.com/user/Kappa_Neuro/models. So yes the repo is not that popular, but lychori are increasingly showing up on Civitai.

**Comment by patrickvonplaten** [comment]
Yes, I think we should try to support it. It's a bit frustrating to constantly see new formats popping up :sweat: - is there any way we can agree on a unified format?

**Comment by patrickvonplaten** [comment]
I think we can solve this with #3756 no?

**Comment by patrickvonplaten** [comment]
cc @sayakpaul

**Comment by cm5kZGV2MjAyM3B1YmdpdGh1YmFjYw** [comment]
> Yes, I think we should try to support it. It's a bit frustrating to constantly see new formats popping up 😓 - is there any way we can agree on a unified format?

True ^_^ ,but i think you probably need to beg community to not research more network designs.

and secondly we will need to 1st agree on some network designs and behaviour.like if the community likes chained LoRA then both needs to agree on it. if both agrees to have the same module implemented then ideally we can even make a shared* repo. which contains those components.

**Comment by cm5kZGV2MjAyM3B1YmdpdGh1YmFjYw** [comment]
bump havent tested v20 if lycoris is supported or not.

**Comment by patrickvonplaten** [comment]
Can someone add a reproducible error/codesnippet here - otherwise we can't help
